src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala

```diff
@@ -126,6 +126,7 @@ trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
   override def write_map(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlMap])
   override def write_object(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlObject])
   override def write_objects(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlArray])
+  override def write_tsv(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlArray])
```

Comment:
heh yeah...  also `write_lines` uses `writeToTsv` too.

`write_lines` - 1 column TSV.
`write_map` - 2 column TSV.  Column 1 is key, column 2 is value
`write_object` - 2 row TSV.  Row 1 is is headers, row 2 are the corresponding values
`write_objects` - 2+ row TSV.  Row 1 is headers, row 2-n are the corresponding values for each object
`write_tsv` - Most general case of serializing any `Array[Array[Primitive]]` as a TSV